src/dotnet-svcutil/lib/tests/src/TestInit.cs

```diff
@@ -117,10 +129,16 @@ public ClassFixture()
 
             var vsTestsRoot = GetSTestRootDir();
 
-            g_SdkVersion = "3.1.100";
+            g_SdkVersion = "5.0.100";
```

Comment:
what happens if we need to run test on version 3.1?

Reply:
When we need to test version 3.1, the 3.1 version is passed via global.json, and it's get at runtime from test code by executing command "dotnet --version", I've validated manually the test passed.